pymchelper/writers/hdf.py

```diff
@@ -0,0 +1,66 @@
+import logging
+import numpy as np
+
+logger = logging.getLogger(__name__)
+
+
+class HdfWriter:
+    """
+    Supports writing HDF file format
```

Comment:
Maybe you could add a few more info what this format is, and what uses it?

Reply:
Done